components/checkbox/flag.js

```diff
@@ -0,0 +1,18 @@
+import h from 'virtual-dom/h';
+
+import Rx from 'rx';
+
+import {labelledCheckbox} from './labelled';
+
+
+const $combine = Rx.Observable.combineLatest;
+
+export function flagCheckbox(label, initialState = false) {
```

Comment:
It was when this was going to be a flag - it's still handy as it constructs the URL:
```
checked    => ?free=true&costModelDiff=true
unchecked  => ?free=true
```